### PR TITLE
UIOR-188 Filtering by status doesn't work for order results

### PR DESCRIPTION
**Pull request by NikitaSedyx**
## Purpose

Fix filtering by status on orders page
https://issues.folio.org/browse/UIOR-188

## Approach

Used correct property and values

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/Utils/FilterConfig.js [hunk 1]
```diff
@@ -7,10 +13,10 @@ const Filters = () => {
...
     },
     {
-      label: 'Approval Status',
-      name: 'approvalStatus',
-      cql: 'approvalStatus',
-      values: ['Approved', 'Pending', 'On hold', 'Not approved'],
+      label: <FormattedMessage id="ui-orders.order.workflow_status" />,
+      name: 'workflowStatus',
+      cql: 'workflowStatus',
+      values: values(WORKFLOW_STATUS),
```
Comment (aliaksei-chumakou) [inline src/components/Utils/FilterConfig.js]: `Object.values(WORKFLOW_STATUS)` ? and no import of 3rd party, what do you think?

Comment (NikitaSedyx) [reply src/components/Utils/FilterConfig.js]: do we have polyfill for  `Object.values`? as it doesn't support all browsers

lodash is already included into the project, why not to use it

Comment (aliaksei-chumakou) [reply src/components/Utils/FilterConfig.js]: according to the https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/values it's not supported only in IE, (OK in edge), I don't think we support IE actually, but for sure it's safer to use lodash's thing

Comment (NikitaSedyx) [reply src/components/Utils/FilterConfig.js]: wow, safari is green, as I remember it didn't previously

**Pull request merged**